test/js/explore/explore-display-tabs.test.js

```diff
@@ -18,6 +18,16 @@ jest.mock('components/visualization/InferCNVIdeogram', () => {
 })
 
 import { getEnabledTabs } from 'components/explore/ExploreDisplayTabs'
+import React from 'react'
+import { render, screen, waitForElementToBeRemoved } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import * as Reach from '@reach/router'
+import jquery from 'jquery'
+
+import MockRouter from '../lib/MockRouter'
+import * as ScpApi from 'lib/scp-api'
+import ExploreView from 'components/explore/ExploreView'
+import { mockPerformance } from '../mock-performance'
```

Comment:
VS Code shows only one of these imports is used, and 

```
yarn test test/js/explore/explore-display-tabs.test.js # Run a particular test suite
```

passes with the suggested change to remove the unused code.

```suggestion
import '@testing-library/jest-dom/extend-expect'
```